package.json

```diff
@@ -6,13 +6,15 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.3.5",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
     "start": "react-scripts start",
+    "server": "json-server -p 3000 --host 127.0.0.1",
```

Comment:
Are we using this? I think this is some kind of in-memory JSON database

Reply:
You are right. This is unnecessary. I will remove it.